regression/tests/studio/test_studio_home.py

```diff
@@ -43,3 +46,44 @@ def test_studio_course_select(self):
         self.studio_home_page.select_course(
             'Manual Smoke Test Course 1 - Auto')
         self.studio_course_outline.wait_for_page()
+
+
+class StudioLmsTest(WebAppTest):
+    """
+    Tests that require lms verification with studio
+    """
+
+    DEMO_COURSE_USER = os.environ.get('USER_LOGIN_EMAIL')
+    DEMO_COURSE_PASSWORD = os.environ.get('USER_LOGIN_PASSWORD')
+
+    def setUp(self):
+        """
+        Initialize the page object
+        """
+        super(StudioLmsTest, self).setUp()
+        # Login to Lms first to avoid authentication
+        self.login_page = LmsLogin(self.browser)
+        LoginHelper.login(self.login_page)
+
+        self.studio_login_page = StudioLogin(self.browser)
+        self.studio_login_page.visit()
+        self.studio_login_page.login(self.DEMO_COURSE_USER,
+                                     self.DEMO_COURSE_PASSWORD)
+        self.studio_home_page = DashboardPageExtended(self.browser)
+
+        self.course_info = {
+            'org': 'ArbiRaees', 'number': 'AR-1000', 'run': 'fall'}
```

Comment:
Same here....this should be a var so it's in fewer places in the code... I also think it makes sense to have it as an environment var